app/assets/stylesheets/views/_brexit_checker_results_page.scss

```diff
@@ -85,28 +85,33 @@
   }
 
   .brexit-checker__print-link {
-    @include govuk-font(19);
-    background: image-url("govuk_publishing_components/icon-print.png") no-repeat 10px 50%;
-    padding: 0.5em 0.5em 0.5em 38px;
-    margin-left: -10px;
-    margin-top: govuk-spacing(3);
-    outline: 0;
-    border: 0;
-    cursor: pointer;
-    text-decoration: underline;
-    color: $govuk-link-colour;
-
-    @include govuk-device-pixel-ratio($ratio: 2) {
-      background-image: image-url("govuk_publishing_components/icon-print-2x.png");
-      background-size: 16px 18px;
-    }
-
-    &:hover {
-      color: $govuk-link-hover-colour;
-    }
-
-    &:focus {
-      @include govuk-focused-text;
+    display: none;
+
+    .js-enabled & {
```

Comment:
Interesting, so what adds/removes this class?

Reply:
https://github.com/alphagov/static/blob/6a0d8a619e56063c81c193458bff92618f163306/app/views/layouts/govuk_template.html.erb#L40